adapter/reaction_event.py

```diff
@@ -0,0 +1,43 @@
+import enum
+
+from typing import Any
+from pydantic import BaseModel, ConfigDict, UUID4
+from bot.bot import MessageAdapter
+
+
+class UnknownReaction(Exception):
+    message = "Unknown reaction found"
+
+
+class Reaction(enum.StrEnum):
+    POSITIVE = "Positive"
+    NEGATIVE = "Negative"
+
+
+class ReactionEventCreate(BaseModel):
+    bot_id: UUID4
+    reaction: Reaction
+    source_adapter: MessageAdapter
+    source_adapter_message_id: str
+    source_adapter_user_id: str
+    message: str
+
+    model_config = ConfigDict(from_attributes=True)
+
+    @staticmethod
+    def from_slack_reaction(bot_id: str, message: str, event: dict[str, Any]):
+        if event["reaction"] == "-1":

```

Comment:
how to convert from emoji into "-1" and "+1"?

Reply:
The `:thumbsdown:` and `:thumbsup:` are represented as `-1` and `+1` respectively from Slack themselves